index.js

```diff
@@ -0,0 +1,327 @@
+'use strict';
+/* eslint new-cap: ["error", { "capIsNewExceptionPattern": "^Sequelize\.." }] */
+/* eslint no-underscore-dangle: ["error", { "allowAfterThis": true }] */
+const Datastore = require('screwdriver-datastore-base');
+const schemas = require('screwdriver-data-schema');
+const Sequelize = require('sequelize');
+const MODELS = schemas.models;
+const MODEL_NAMES = Object.keys(MODELS);
+
+/**
+ * Converts data from the value stored in the datastore
+ * @method decodeFromDialect
+ * @param  {String}          dialect Underlying system that we're reading from
+ * @param  {SequelizeRow}    content Row that Sequelize returns to us
+ * @param  {Object}          model   Screwdriver Data Schema about the Model
+ * @return {Promise}                 Decoded Values (object)
+ */
+function decodeFromDialect(dialect, content, model) {
```

Comment:
So `decode` parse the values if they are array or object. And `encode` turn them to string so we can safely store them into the datastore. Is that right?